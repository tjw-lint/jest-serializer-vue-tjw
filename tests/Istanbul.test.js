const helpers = require('./helpers.js');

import { mount } from '@vue/test-utils';
import Instanbul from './components/Instanbul.vue';

describe('Instanbul.vue', () => {
  test('Comment removed', () => {
    helpers.mockSettings({
      removeIstanbulComments: true
    });
    let wrapper = mount(Instanbul);

    expect(wrapper)
      .toMatchSnapshot();
  });
});
