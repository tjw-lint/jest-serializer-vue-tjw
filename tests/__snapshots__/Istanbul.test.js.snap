// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Istanbul.vue Comment removed 1`] = `
<div title="function () {
        return true;
      }">
  <section>
    <article class="alright">
      <aside>
        <h1 class="okay">okay</h1>
        <fieldset>
          <form>
            <label for="text" stuff="function (x) {
        return !x;
      }">
              Text
            </label>
            <input aria-label="text" aria-required="true" id="text" requried thing="function () { [native code] }">
          </form>
        </fieldset>
      </aside>
    </article>
  </section>
</div>
`;
